### ISSUE TITLE
Small bug in Showo.mmu_generate？

### ISSUE THREAD

**Issue by CladernyJorn** [showlab/Show-o#49]
In line 221  of file .\models\modeling_showo.py, probably the shape of attention (L+1,L+1) instead of (1,1,L+1,L+1) mask is changed while iteration of generating new words. It can be fixed by adding codes like 'attention_mask = attention_mask_b.unsqueeze(0).unsqueeze(0)'.

**Comment by Sierkinhane** [comment]
Hi, I'm not sure if you encounter errors when using the current version.

**Comment by CladernyJorn** [comment]
I encountered an error when using the current version for mmu: Attention mask should be of size (1, 1, 621, 621), but is torch.Size([1, 1, 620, 620]).
